Mods/BreathOfTheWild_Intel Remove Bloom/rules.txt

```diff
@@ -0,0 +1,6 @@
+[Definition]
+titleIds = 00050000101C9300,00050000101C9400,00050000101C9500
+name = "Remove Intel Excess Bloom"

```

Comment:
Should change this to "Fix Excessive Bloom On Older Intel iGPU Drivers".

Reply:
Name modified as suggested